src/app/static-pages/landing-page/sections/supporter-section/supporter-section.component.html

```diff
@@ -1,4 +1,4 @@
-<p class="title">codetekt wird unterstützt von</p>
+<p class="title">Unsere Partner:innen, Unterstützer:innen und Netzwerke</p>
```

Comment:
Haben wir eine einheitliche Art fürs Gendern - glaube sonst wird * verwendet, oder? 🤨

Reply:
@SandroGlueck Bisher ist es mal so, mal so. Denke das ist was für @Th3da und die Redaktion, da mal einheitliche Regeln aufzustellen und dann auch anzuwenden.

Reply:
Heikles Thema. 
Ein Text wie dieser 
`Ist es z.B. ein*e bekannte*r Journalist*in, Autor*in oder ein*e anerkannte*r Wissenschaftler*in?`
ist pc, aber nicht besonders gut lesbar.

Reply:
Dass wir prinzipiell geschlechtergerechte Sprache verwenden wollen, steht soweit ich das sehe nicht zur Debatte. 
Die Frage, die sich stellt, ist eher die Umsetzung. An einigen Stellen, lassen sich schlechter lesbare Sternchen oder Doppelpunkte vermeiden, indem wir auf das Passiv umsteigen (Nutzende statt Nutzer*innen). 
Für die Fälle, in denen es nicht anders geht, sollten wir uns aber mE trotzdem überlegen, welches Trennzeichen wir verwenden wollen und das dann konsequent durchziehen.

Reply:
So hatte ich es doch  gemeint: es geht um die Umsetzung - aber die ist nicht einfach.

Aufgefallen war mir der Text, weil ich gerade die review_questions.json bearbeitet hatte (Anlass waren eigentlich nur die Typos, dann habe ich mich etwas intensiver damit beschäftigt).

```
"content": "Der Artikel ist von einer fachkundigen Person oder einer*m beruflichen Journalist*in geschrieben.",
"info": "Die Veröffentlichung von Inhalten unter dem eigenen Namen ermöglicht es, Inhalte zu einer bestimmten existierenden Person zurückzuführen und diese neispielsweise für eine Stellungnahme zu kontaktieren. Es ist ein Indiz für professionnell erstellte Inhalte.",
"hint": "Oft findest du diese Information entweder unter dem Titel oder unter dem Artikel. Ist es z.B. ein*e bekannte*r Journalist*in, Autor*in oder ein*e anerkannte*r Wissenschaftler*in? Ist es eine Person, die du aus dem öffentlichen Leben kennst? ",
```
Es gibt außer den (einheitlich zu verwendenden) Gender-Trennzeichen noch die Möglichkeit, Sätze so zu gestalten, dass die Gender-Differenzierung möglichst zurückhaltend wird. Wir haben im Deutschen das Problem, dass es nicht reicht, Nomen geschlechtergerecht zu markieren, sondern auch Pronomen und Adjektive - dazu noch alle flektierten Formen.
Deshalb könnten wir auch überlegen, ob eine alternative Formulierung möglich ist - also schon im Ansatz inklusiver zu schreiben. 
Zum Beispiel:

```
"content": "Der Artikel wurde von einer Person geschrieben, die berufliche Expertise hat oder professionell journalistisch tätig ist.",
"info": "Die Veröffentlichung von Inhalten unter dem eigenen Namen ermöglicht es, die Aussagen einer existierenden Person zuzuordnen und 
diese gegebenenfalls für eine Stellungnahme zu kontaktieren. Es ist ein Indiz für professionell erstellte Inhalte.",
"hint": "Oft findest du diese Information entweder unterhalb vom Titel oder am Ende des Artikels. Findest Du die Person unter den bekannten Journalist*innen, Autor*innen oder anerkannten Wissenschaftler*innen, ist sie aus dem öffentlichen Leben bekannt? ",

Reduktion der Sternchen von 9 auf 3 - der Text ist sogar sprechbar.
```

Reply:
Perfekt! Deinen Vorschlag finde ich super! :)